client/server/server.ts

```diff
@@ -0,0 +1,11 @@
+import { GET_ALL_MARKETS_QUERY, GET_ALL_ENCRYPTED_MARKETS_QUERY } from "./silicon/silicon_query";
+import { Silicon, instantiate_silicon } from "./silicon/silicon";
+import { verifySiliconMapping } from "./silicon/silicon_utils"; 
+import { EncryptedMarket, Market } from "../graphql/graphql";
+
+(async () => {
+    const transparent_silicon = await instantiate_silicon(GET_ALL_MARKETS_QUERY) as Silicon<Market>;
+    const encrypted_silicon = await instantiate_silicon(GET_ALL_ENCRYPTED_MARKETS_QUERY) as Silicon<EncryptedMarket>;
+    const isValidMapping =  verifySiliconMapping(encrypted_silicon, transparent_silicon);
```

Comment:
Are you keeping both markets up here for dev purposes?

Reply:
yes, haven't wrapped my head around that bit yet, so left it there... if you're seeing something I don't yet, let , me know !